### ISSUE TITLE
Watch mode failure

### ISSUE THREAD

**Issue by cgatian** [dotansimha/graphql-code-generator#2138]
**Describe the bug**
Watch mode does not work. This is due to an issue with chokidar.

```bash
(node:87504) UnhandledPromiseRejectionWarning: TypeError: Cannot mix BigInt and other types, use explicit conversions
    at FSWatcher._hasReadPermissions (M:\repositories\_trash\ng-graphql-sample\node_modules\@graphql-codegen\cli\node_modules\chokidar\index.js:775:36)
    at WatchHelper.filterPath (M:\repositories\_trash\ng-graphql-sample\node_modules\@graphql-codegen\cli\node_modules\chokidar\index.js:191:16)
    at ReaddirpStream.fileFilter [as _fileFilter] (M:\repositories\_trash\ng-graphql-sample\node_modules\@graphql-codegen\cli\node_modules\chokidar\lib\nodefs-handler.js:405:29)
    at ReaddirpStream._isFileAndMatchesFilter (M:\repositories\_trash\ng-graphql-sample\node_modules\readdirp\index.js:234:31)
    at ReaddirpStream._exploreDirectory (M:\repositories\_trash\ng-graphql-sample\node_modules\readdirp\index.js:163:23)
(node:87504) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:87504) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

```

**To Reproduce**
Add watch flag on Windows with the latest version (1.3.1)

**Expected behavior**
No error

**Environment:**
- OS: Windows 10
- NodeJS: 10.16.0

**Additional context**
Using the latest canary build of the CLI works. This issue is related to a bump to the chokidar 3.0.2. Adding this issue for anyone else that may encounter this hard to debug issue.

https://github.com/paulmillr/chokidar/releases/tag/3.0.2

**Comment by cgatian** [comment]
https://github.com/paulmillr/chokidar/issues/868

**Comment by dotansimha** [comment]
Fixed in 1.4.0 🎉
